components/language-switcher/language-switcher.html

```diff
@@ -0,0 +1,33 @@
+<form class="mzp-c-language-switcher " method="get"
+ action="#">
```

Comment:
There are some awkward line breaks in the markup, I assume copied from how the code is formatted on https://protocol.mozilla.org/patterns/molecules/language-switcher.html. But we can tolerate longer lines than whatever that formatter is set for, so this could be tidied up a bit. You might prefer to copy from the original sources (https://github.com/mozilla/protocol/blob/main/src/patterns/molecules/language-switcher/language-switcher.hbs) and convert the Handlebars to Nunjucks.